File: 6KYU/Multiples of 3 or 5/solution.cpp
int solution(int n) 
{
    int total = 0;
  
    for (int i=1; i<n; i++) { 
      if(i % 3 == 0 || i % 5 == 0){
        total = total + i;
      }
    }
    return total;
}

File: 6KYU/Multiples of 3 or 5/solution.jl
module Solution
export sumofmultiples

  function sumofmultiples(number)
     total = 0

     for i in 0:number-1
         if i % 3 == 0 || i % 5 == 0
           total = total + i
         end
     end
     return total
  end

end